examples/asr/conf/hat/conformer/conformer_hat_bpe.yaml

```diff
@@ -0,0 +1,267 @@
+# It contains the default values for training a Conformer-Hybrid_Autoregressive_Transducer ASR model, large size (~120M) with Transducer loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+# Here are the recommended configs for different variants of Conformer-Transducer, other parameters are the same as in this config file.
+#
+#  +--------------+---------+---------+----------+------------------+--------------+--------------------------+-----------------+
```

Comment:
Are all the hyper parameters tuned to give strong HAT model? Please setup the default config for Conformer HAT Large to get best results on LS / ASRSET

Reply:
I did not try different hyper parameters for HAT model. I got comparable results (in greedy decoding) for HAT and RNNT models with the same parameters. I assume that default RNNT hyper parameters are suitable for HAT model as well.

Reply:
Ok sounds good